README.md

```diff
@@ -325,6 +325,19 @@ You can login to test features like posting, replying, tipping, etc with `./snde
 
 ### Login with Email
 
+#### MailHog
+
+- The app is already prepared to send emails through [MailHog](https://github.com/mailhog/MailHog) so no extra configuration is needed
+- Click "sign up" and enter any email address (remember, it's not going anywhere beyond your workstation)
+- Access MailHog's web UI on http://localhost:1025
```

Comment:
I think you mean to say port `8025`.

`1025` is the smtp port.

Reply:
oops. Fixed!

Now I'm going to fix the healthcheck